guide.md

```diff
@@ -259,7 +260,7 @@ This is a _bare minimum_ implementation that demonstrates how to access REDCap v
 
 This module introduces you to interactions with the `redcap_data` table. You will finish a page that will allow a user with admin rights to insert an arbitrary text value into a field across all records of a project. You should look at the documentation for the `getData()` and `saveData()` methods on your REDCap instance under **Control Center -> Plugin, Hook, & External Module Documentation**.
 
-This module uses AJAX. Your updates should be made in `pages/ajaxpage.php`. You should still look through the other files to understand the module as a whole.
+This module uses AJAX. Your updates should be made in `pages/ajaxpage.php`. Be sure to review the methods in `RecordWranglingModule.php` as much of the business logic is already written. 
```

Comment:
I added this not because I didn't see that so much of the business logic was actually already written.  I was prototyping a lot of the logic on the page before I saw much of it was already written.